README.md

```diff
@@ -287,53 +287,161 @@ Although this example is pretty simple and not even includes Lens composition, i
 
 Just as with React, but assuming Angular instead :)
 
+The code for below example - can be found here <https://github.com/georgebatalinski/telescope_todo>
+
 As a bootstrap for the application we need something like:
 
+```typescript 
+@NgModule({
+  declarations: [
+    AppComponent,
+    TodoListComponent
+  ],
+  imports: [
+    ...
+  ],
+  providers: [
+    { provide: 'StreamContract', useFactory: ()=> Telescope.of( AppState.empty() ), deps: [] }
+  ],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+
+We define our global STATE (however, the more ultimate approach would be to have local state)
+local state = would mean each Service, would have a Telescope instance of their own, resulting in us not 
```

Comment:
Consider removing the `=` symbol and write down in words what you want to say .

Reply:
Sounds good , I will update this